Automattic/jetpack
# Build / JITM: move assets inside package and build all CSS when building plugin

Fixes #13089

#### Changes proposed in this Pull Request:

* We will now ship a JITM package (but also all other DNA packages) with an `assets` folder that will contain all necessary CSS files (minified as well as RTL files) for that package.

**Note**: I did not include .map files. Should we aim to include those?

#### Testing instructions:

* Start from a site running WordPress in Arab, or Hebrew
* Install and connect Jetpack with this branch.
* Once connected to WordPress.com, look for pages that will display JITMs. Dashboard > Updates should show one, as well as the main Dashboard, or Appearance > Themes in some cases.
* The JITMs should be displayed properly.

#### Proposed changelog entry for your changes:

* Dashboard Notices: fix layout issues on sites using an RTL language.
